ScanCommand/src/main/java/io/ballerina/scan/ScanCmd.java

```diff
@@ -139,6 +184,29 @@ private StringBuilder helpMessage() {
         return builder;
     }
 
+    private Project getProject() {
+        if (argList.isEmpty()) {
+            try {
+                return BuildProject.load(Paths.get(System.getProperty(ProjectConstants.USER_DIR)));
+            } catch (RuntimeException ex) {
+                outputStream.println(ex.getMessage());
+                return null;
+            }
+        } else {
+            Path path = Paths.get(argList.get(0));
+            try {
+                if (path.toFile().isDirectory()) {
+                    return BuildProject.load(path);
+                } else {
+                    return SingleFileProject.load(Paths.get(argList.get(0)));
+                }
+            } catch (RuntimeException ex) {
+                outputStream.println(ex.getMessage());
+                return null;
+            }
+        }
```

Comment:
Let's remove `else` statement when it is not necessary

```suggestion
        }
        Path path = Paths.get(argList.get(0));
        try {
            if (path.toFile().isDirectory()) {
                return BuildProject.load(path);
            } 
            return SingleFileProject.load(Paths.get(argList.get(0)));
        } catch (RuntimeException ex) {
            outputStream.println(ex.getMessage());
            return null;
        }
```